src/extractor/guidance/motorway_handler.cpp

```diff
@@ -490,7 +490,7 @@ Intersection MotorwayHandler::fromRamp(const EdgeID via_eid, Intersection inters
     else
     { // FALLBACK, hopefully this should never been reached
         util::Log(logDEBUG) << "Reached fallback on motorway ramp with " << intersection.size()
```

Comment:
heads up: these loggings have just been removed, since we don't need to check them anymore. Only extreme cases that we don't really care about. We don't need to clutter our logs for these